packages/core-mobile/app/store/balance/slice.ts

```diff
@@ -194,13 +195,15 @@ export const selectTokensWithBalanceForAccount = createSelector(
 )
 
 export const selectBalanceTotalInCurrencyForAccount =
-  (accountIndex: number) => (state: RootState) => {
+  (accountIndex: number, blacklist: LocalTokenId[]) => (state: RootState) => {
     const tokens = selectTokensWithBalanceForAccount(state, accountIndex)
 
-    return tokens.reduce((total, token) => {
-      total += token.balanceInCurrency ?? 0
-      return total
-    }, 0)
+    return tokens
```

Comment:
This is probably producing unoptimised selector warning since for same input new array is always generated.